packages/core/src/types/mistral.d.ts

```diff
@@ -1,3 +0,0 @@
-declare module "@mistralai/mistralai" {
```

Comment:
@marcusschiesser I think they fixed their TSDs, but can you verify that this is no longer needed?

Reply:
yes, looks good, i run our mistral example